### DISCUSSION TITLE
Honk Pending PR List Summary

### DISCUSSION THREAD

**Discussion by karolinajoksaite** [oneroar/discussions#12] [Feature Requests]
It would be really nice to have a summary of pending PRs that can be sent to channels, with configurable frequency (daily, weekly,  etc.).

Essentially instating this functionality:
<img width="532" alt="Screenshot 2024-01-24 at 08 49 59" src="https://github.com/oneroar/discussions/assets/122898962/07415dee-0322-4829-83c5-c2c9a9f66655">

**Comment by DragosDumitrache** [comment]
Hello. I've got something in progress for this, I'll let you know as soon as it can be tried

**Reply by karolinajoksaite** [reply]
Cool!! Thanks

**Comment by dominikrys** [comment]
Hey @DragosDumitrache! Thanks for adding this functionality - we just had it trigger this morning and it's already proved useful!

We did find that many of the PRs that were reported to still be open have already been closed or merged, though. Are you aware of this issue?

**Reply by DragosDumitrache** [reply]
Yep, I noticed that, will put in a fix for it shortly! Thank you for flagging though
